tests/sentry/utils/performance_issues/test_n_plus_one_api_calls_detector_extended.py

```diff
@@ -177,12 +177,12 @@ def test_detects_problems_with_low_span_duration_high_total_duration(self):
 
     def test_does_not_detect_problems_with_low_span_count(self):
         event = get_event("n-plus-one-api-calls/n-plus-one-api-calls-in-issue-stream")
-        event["spans"] = self.create_eligible_spans(1000, 5)  # count is 5, greater than default
+        event["spans"] = self.create_eligible_spans(1000, 10)  # count is 5, greater than default
```

Comment:
Ditto, comment seems outdated

Reply:
By the way a trick to make this comment unnecessary might be: instead of hard-coding the count, make it `minimum_span_threshold - 1` (reading that min span threshold from settings). That way the code directly communicates that the span count is less than the threshold, and still works even if the threshold changes.